test/valid-seq-json/all-possible-fields.seq.json

```diff
@@ -254,11 +254,40 @@
           "time": { "tag": "00:00:01.000", "type": "COMMAND_RELATIVE" },
           "type": "command",
           "args": []
+        },
+        {
+          "description": "Note step with all possible fields.",
+          "models": [
```

Comment:
What do these models do in the context of a `note`? Can the note reference these values?

Reply:
The note itself can't reference the models, but the note can be used as an anchor where models are updated. For example, someone may know that some telemetry value is going to be updated to some at some absolute time (not relative to anything else) and they need that value to be updated at that time. They can place a note at that absolute time and attach models to the note to update the telemetry states.